packages/aws-cdk-lib/aws-ec2/lib/instance-types.ts

```diff
@@ -611,20 +611,30 @@ export enum InstanceClass {
    */
   STORAGE_COMPUTE_1 = 'storage-compute-1',
 
+  /**
+   * Storage/compute balanced instances, 1st generation
+   */
+  H1 = 'h1',
+
   /**
    * High performance computing powered by AWS Trainium
    */
-  TRN1 = 'trn1',
+  TRAINING_ACCELERATOR1 = 'training-accelerator1',
 
   /**
    * High performance computing powered by AWS Trainium
    */
-  TRN1N = 'trn1n',
+  TRN1 = 'trn1',
 
   /**
-   * Storage/compute balanced instances, 1st generation
+   * Network-optimized high performance computing powered by AWS Trainium
    */
-  H1 = 'h1',
+  TRAINING_ACCELERATOR1_ENHANCED_NETWORK = 'training-accelerator1-enhanced-network',
+
+  /**
+   * Network-optimized high performance computing powered by AWS Trainium
+   */
+  TRN1N = 'trn1n',
```

Comment:
Description for this has changed.

Reply:
Yeah, the `trn1n` description didn't have the network optimization discriminator, both `trn1*` had the same description: https://github.com/aws/aws-cdk/blob/fac4a9c23f8e9090b3dc7e26a8306d3a8034b4c9/packages/aws-cdk-lib/aws-ec2/lib/instance-types.ts#L614-L622

Reply:
Gotcha, thanks for clarifying